lib/puppet_x/msutter/templates/set_dsc_configuration.erb

```diff
@@ -10,6 +11,10 @@ configuration PuppetConfiguration
 <%   if resource.parameters[:dscmeta_module_name] && resource[:dscmeta_import_resource] -%>
     Import-DscResource <%= "-Name #{resource[:dscmeta_resource_name]}" if resource.parameters[:dscmeta_resource_name] %> <%= "-ModuleName #{resource[:dscmeta_module_name]}" if resource.parameters[:dscmeta_module_name] %>
 <%   end -%>
+<% else  -%>
+    if ($Posh5OrGreater ) {
+        Import-DscResource -ModuleName PSDesiredStateConfiguration
+    }
```

Comment:
We have other fixes that we've just determined don't require the guarding at all. So yay for less stuff!